Inc/firmware/configuration.hpp

```diff
@@ -53,6 +53,10 @@ constexpr uint8_t BATTERY_PUB_PERIOD = 10;
 constexpr uint8_t JOINTS_PUB_PERIOD = 5;
 constexpr uint8_t ODOM_PUB_PERIOD = 5;
 constexpr uint8_t IMU_PUB_PERIOD = 1;
+constexpr uint16_t PARAM_TRIGGER_PUB_PERIOD = 500;
```

Comment:
I think we can decrease the param trigger pub period to 1s (value of 100)

Reply:
Done